src/lib/webpack-isomorphic-tools-configuration.js

```diff
@@ -36,13 +36,14 @@ let userExtensions = [];
 module.exports = {
   assets: {
     images: {
-      extensions: ["png", "jpg", "gif", "ico", "svg"]
+      extensions: ["png", "jpg", "gif", "ico", "svg"],
+      regular_expression: /\.(png|jpg|gif|ico|svg)(\?v=\d+\.\d+\.\d+)?$/
```

Comment:
Do you think this regex is better than `/\.(png|jpg|gif|ico|svg)(\?v=\d+\.\d+\.\d+)(\?.*)?$/`? Yours will work for font awesome but what if another library has something else after the question mark? Maybe mine is too broad, what do you think?